### PR TITLE
release fixes

### PR DESCRIPTION
**Pull request by nalinbhardwaj**
- [x] fix too slow delay on autofocus
- [x] disable bounciness on Home Screen bottom sheet

### CODE REVIEW

File: apps/daimo-mobile/src/sync/recipients.ts [hunk 1]
```diff
@@ -86,7 +86,7 @@ export function useRecipientSearch(account: Account, prefix: string) {
 
   return {
     isSearching: enabled,
-    recipients,
+    recipients: recipients.slice(0, 16),
```
Comment (dcposch) [inline apps/daimo-mobile/src/sync/recipients.ts:89]: 👍

File: apps/daimo-mobile/src/view/screen/receive/ReceiveScreen.tsx [hunk 2]
```diff
@@ -47,30 +51,15 @@ function RequestScreenInner({
...
-      clearTimeout(focusTimeout);
-    }
-  }, [isFocused, autoFocus]);
-
+  const isFocused = useIsFocused();
+  const nav = useNav();
   const textInputRef = useRef<TextInput>(null);
 
+  // Work around react-navigation autofocus bug
+  useFocusOnScreenTransitionEnd(textInputRef, nav, isFocused, autoFocus);
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/screen/receive/ReceiveScreen.tsx:61]: nice cleanup

File: apps/daimo-mobile/src/view/shared/nav.ts [hunk 3]
```diff
@@ -142,3 +142,30 @@ export function useDisableTabSwipe(nav: MainNav) {
...
+  useEffect(() => {
+    const unsubscribe = nav.addListener("transitionEnd", () => {
+      if (isFocused && autoFocus) {
+        ref.current?.focus();
+
+        // Now, wipe the autoFocus flag so that switching tab and coming back
+        // doesn't keep focusing the input.
+        const { setParams }: { setParams: any } = nav;
+        setParams({
+          autoFocus: false,
```
[the remark this answers is not in the document]
Comment (dcposch) [reply apps/daimo-mobile/src/view/shared/nav.ts:164]: if you want to only call if it exists:

```ts
(nav as any).setParams?.({ autoFocus: false });
```

**Review by dcposch — APPROVED** [review]

File: apps/daimo-mobile/src/view/shared/nav.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (nalinbhardwaj) [reply apps/daimo-mobile/src/view/shared/nav.ts]: done

**Pull request merged**